src/components/search/SearchSection.jsx

```diff
@@ -1,35 +1,81 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect, useContext } from 'react';
+import axios from 'axios';
 import { FaSearch, FaMapMarkerAlt, FaRegCalendar } from 'react-icons/fa';
 import DatePicker from 'react-datepicker';
-import AutoComplete from './Autocomplete';
+// import AutoComplete from './Autocomplete';
 import 'react-datepicker/dist/react-datepicker.css';
 import styles from '../../styles/Search.module.css';
+import SearchContext from './SearchContext';
 
 const SearchSection = () => {
   const dummyTags = ['Starting soon', 'Tomorrow', 'Near you', 'Dancers', 'Clowns', 'Magicians'];
+  const geoLocation = { lat: 29.954767355989652, lng: -90.06911208674771 };
   const [searchTerm, setSearchTerm] = useState('');
-  const [searchLocation, setSearchLocation] = useState('');
+  const [searchLocation, setSearchLocation] = useState(geoLocation);
   const [searchDate, setSearchDate] = useState('');
+  const { setResults } = useContext(SearchContext);
+
+  const onSearchSubmit = () => {
+    axios.get('https://www.buskr.life/api/events', {
+      params: {
+        features: 'coords,location,photos,tags',
+        lat: searchLocation.lat,
+        lng: searchLocation.lng,
+        from: searchDate
+          || new Date(),
+        limit: 100,
+      },
+    }).then((result) => {
+      const byTime = result.data.features.sort(
```

Comment:
`sort` mutates its caller. In other words, what you're really doing is
1. Sort `result.data.features` by time.
2. Create another variable called `byTime` that points to the same object as `result.data.features`.

I think that is probably not what you're trying to do. Maybe you should call `result.data.features.slice().sort` instead?